scriptmodules/emulators/quasi88/01_Makefile.diff

```diff
@@ -0,0 +1,98 @@
+diff -u quasi88-0.6.4_org/Makefile quasi88-0.6.4/Makefile
+--- quasi88-0.6.4_org/Makefile	2013-02-19 14:54:03.000000000 +0000
++++ quasi88-0.6.4/Makefile	2017-01-29 04:51:05.264262542 +0000
+@@ -16,8 +16,8 @@
+ #	GTK�Ǥξ��ϡ���GTK_VERSION�װʳ��ιԤ򥳥��ȥ����Ȥ��Ƥ��������
+ #	( GTK�Ǥϼ¸���Ǥ���GTK�Ǥǥ�����ɤ��Ĥ餹�ˤϡ�SDL ��ɬ�פǤ� )
+ 
+-X11_VERSION	= 1
+-# SDL_VERSION	= 1
++# X11_VERSION	= 1
++SDL_VERSION	= 1
+ # GTK_VERSION	= 1
+ 
+ 
+@@ -32,7 +32,7 @@
+ #	�ǥ��쥯�ȥ꤬���Ѥ���ޤ���
+ #	 ~/ �ϡ�QUASI88�ε�ư���˥ۡ���ǥ��쥯�ȥ��$HOME�ˤ�Ÿ������ޤ���
+ 
+-ROMDIR	= ~/quasi88/rom/
++ROMDIR	= /home/pi/RetroPie/BIOS/quasi88
```

Comment:
This shouldn't be hardcoded as RetroPie can be used for any user.

You can put the patch in-line and use romdir/BIOS/quasi88, or put some string here and sed the patch before applying.